pkgs/development/compilers/hip/default.nix

```diff
@@ -172,6 +169,14 @@ stdenv.mkDerivation (finalAttrs: {
     #!/usr/bin/env nix-shell
     #!nix-shell -i bash -p curl jq common-updater-scripts nix-prefetch-github
     version="$(curl ''${GITHUB_TOKEN:+"-u \":$GITHUB_TOKEN\""} -sL "https://api.github.com/repos/ROCm-Developer-Tools/HIP/tags" | jq '.[].name | split("-") | .[1] | select( . != null )' --raw-output | sort -n | tail -1)"
+
+    IFS='.'
+    read -a version_arr <<< "$version"
+
+    if [ "''${#version_arr[*]}" == 2 ]; then
+      version="''${version}.0"
+    fi
```

Comment:
We probably want to restore IFS for the rest of the build

Reply:
AFAIK putting IFS before the command should work, but my bash is rusty so let me know if that isn't working.